packages/tsconfig.options.json

```diff
@@ -16,7 +16,7 @@
     "resolveJsonModule": true,
     "sourceMap": true,
     "strict": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
The reason for this change is that TS adds polyfills for `Class` and `Map` if target is 'es5'. The polyfills increases runtime cost and bundle size. 

Looking into other Atlassian owned libraries, [target](https://github.com/atlassian-labs/react-resource-router/blob/master/tsconfig.json#L16) has been set above es5. Therefore, I think it's safe to change target in Compiled as well.

Reply:
You might want to make another changeset with a minor bump that includes this change. Any packages that have a different distributable should have it otherwise we'll break semver

Reply:
oh, good point!